Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  LoginView.swift
+//  Survey
+//
+//  Created by Khanh on 27/07/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+
+struct LoginView: View {
+
+    private var input: LoginViewModel.Input
+    @ObservedObject var output: LoginViewModel.Output
+
+    private let loadTrigger = PassthroughSubject<Void, Never>()
+
+    @State private var image: UIImage? = UIImage(named: "ic_background")
+    @State private var email = ""
+    @State private var password = ""
+
+    private var loginTitle = "Login"
+
+    private let gradient = LinearGradient(
+        gradient: Gradient(stops: [
+            .init(color: .black, location: 0.0),
+            .init(color: .black.opacity(0.2), location: 1.0)
+        ]),
+        startPoint: .bottom,
+        endPoint: .top
+    )
+
+    var body: some View {
+        GeometryReader { geo in
+            ZStack {
+                backgroundSetup()
+                VStack {
+                    logoSetup()
+                        .frame(height: geo.size.height * (1.0 / 3.0))
+                    componentSetup()
+                        .frame(height: geo.size.height * (1.0 / 3.0))
+                    Text("")
+                        .frame(height: geo.size.height * (1.0 / 3.0))
+                }
+            }
+        }
+        .preferredColorScheme(.dark)
+    }
+
+    init(viewModel: LoginViewModel) {
+        let input = LoginViewModel.Input(loadTrigger: loadTrigger.asDriver())
+        output = viewModel.transform(input)
+        self.input = input
+    }
+
+    private func backgroundSetup() -> some View {
+        Image("ic_background")
```

Comment:
We're using `ic_background` image in many place. Should we declare it as a constant and reuse it?

Reply:
Yess, I took advantage of R.Swift to reuse it, fixed at https://github.com/nkhanh44/ic-survey-ios/commit/45ae841b35aa065645f32fb9c3e9b222f1e64540, is this solution ok ?